metrictank.go

```diff
@@ -367,25 +376,19 @@ func main() {
 
 	usg := usage.New(accountingPeriod, metrics, metricIndex, clock.New())
 
-	handlers := make([]mdata.ClusterHandler, 0)
-	if clKafka.Enabled {
-		clKafkaInst = mdata.NewKafka(*instance, metrics, stats)
-		handlers = append(handlers, clKafkaInst)
+	handlers := make([]mdata.NotifierHandler, 0)
+	if notifierKafka.Enabled {
+		notifierKafkaInst = notifierKafka.NewNotifierKafka(*instance, metrics, stats)
+		handlers = append(handlers, notifierKafkaInst)
 	}
 
-	mdata.InitCluster(stats, handlers...)
-
-	if inCarbon.Enabled {
-		inCarbonInst.Start(metrics, metricIndex, usg)
-	}
+	mdata.InitPersistNotifier(stats, handlers...)
 
-	if inKafkaMdm.Enabled {
-		sarama.Logger = l.New(os.Stdout, "[Sarama] ", l.LstdFlags)
-		inKafkaMdmInst.Start(metrics, metricIndex, usg)
-	}
-	if inKafkaMdam.Enabled {
-		sarama.Logger = l.New(os.Stdout, "[Sarama] ", l.LstdFlags)
-		inKafkaMdamInst.Start(metrics, metricIndex, usg)
+	/***********************************
+		Start our receivers
```

Comment:
we have a naming problem:
* in the docs and configs, we talk about inputs
* in the code, we use `in`
* in the main scope, we call them receivers

We should make this consistent to 1 terminology.
`in` is a bit too short to be a meaningful term in docs etc, but `input` is shorter than `receiver`, so I think we should call them input in all 3 places.